### ISSUE TITLE
Tornado 5 breaks the library in notebooks

### ISSUE THREAD

**Issue by aa7955** [IBM-Cloud/sql-query-clients#14]
Upgrading and installing ibmcloudsql now gives the following error. Not sure if it's because of Tornado 5 or something else. A lot of people appear to have issues with notebooks and Tornado 5. Might want to reproduce:

```
!pip install ibmcloudsql # using version 0.2.5
import ibmcloudsql
```

```
ImportError                               Traceback (most recent call last)
<ipython-input-2-6594b185a2ba> in <module>()
----> 1 import ibmcloudsql

~/.local/lib/python3.5/site-packages/ibmcloudsql/__init__.py in <module>()
     15 # ------------------------------------------------------------------------------
     16 
---> 17 from .SQLQuery import SQLQuery

~/.local/lib/python3.5/site-packages/ibmcloudsql/SQLQuery.py in <module>()
     21 import xml.etree.ElementTree as ET
     22 from tornado.escape import json_decode
---> 23 from tornado.httpclient import HTTPClient, HTTPError
     24 from tornado.httputil import HTTPHeaders
     25 import sys

~/.local/lib/python3.5/site-packages/tornado/httpclient.py in <module>()
     45 import weakref
     46 
---> 47 from tornado.concurrent import Future, future_set_result_unless_cancelled
     48 from tornado.escape import utf8, native_str
     49 from tornado import gen, httputil, stack_context

ImportError: cannot import name 'future_set_result_unless_cancelled'
```

Here are the Tornado updates which address the problem apparently: http://www.tornadoweb.org/en/stable/releases/v5.0.0.html

One way around it was to just downgrade Tornado to 4.5.0, but I don't think this solution is sustainable. Any way we could make 4.5.0 the default for the package until there is a solution?

**Comment by aa7955** [comment]
Lots of upgrades to Jupyter that were needed to fix this issue. Perhaps on the DSX side there needs to be some fixes but on Anaconda you just run an update locally.

**Comment by torsstei** [comment]
@aa7955 Is this something that others are likely to hit too?

**Comment by aa7955** [comment]
@torsstei I think it was DSX being slow to update. Also, if users don't update their local environment, like they would have to in Anaconda, they'll run into some issues.

**Comment by torsstei** [comment]
@aa7955 I reproduced it today multiple times. It seems that this occurs only when you `!pip install ibmcloudsql --upgrade`. When not using `--upgrade`it doesn't seem to occur. I don't understand what's screwing up in DSX yet.

**Comment by aa7955** [comment]
@torsstei interesting indeed. I'll see if it's also a problem with Anaconda locally. I'm at a conference, so I'll check between sessions.

**Comment by torsstei** [comment]
@aa7955 The issue is caused by DSX internally depending on tornado 4.5.2 and keeping it around even though you install 5.0. I've limited the prerequed version in setup.py now to 4.5.2 and now the problem is gone even when you install ibmcloudsql with --upgrade. Published as version 0.2.7.

**Comment by aa7955** [comment]
Good to know. 👍 Thanks @torsstei
